test/performance/idpay/idpayOnboardingAPI.js

```diff
@@ -38,6 +39,8 @@ const testName = 'idpayOnboardingAPI'
 // Set up data for processing, share data among VUs
 const cfList = new SharedArray('cfList', getFCList)
 
+ONBOARDING_API_NAMES.getWalletDetail = WALLET_API_NAMES.getWalletDetail;
```

Comment:
evenif it will work, this is formerly wrong because you are changing the original object.
You have to build an array with the apiNames:
`Object.values(ONBOARDING_API_NAMES).concat(WALLET_API_NAMES.getWalletDetail)`